src/Movepiece.js

```diff
@@ -0,0 +1,55 @@
+let piecePosition = {x: 4, y: 0}
+function MovePiece(direction){ // function that moves the piece
+    switch (direction) {
+        case 'left':
+            piecePosition.x -= 1;
+            break;
+        case 'right':
+            piecePosition.x += 1;
+            break;
+        case 'down':
+            piecePosition.y += 1;
+            break; 
+    }
+}
+
+function collision(){ // checks if pieces are touching and stops them from colliding
```

Comment:
With a utility function like this (is this piece colliding with any existing piece in this grid), it's probably best to pass the inputs in explicitly. This makes it much more clear what the function handles & which variables come from outside the function.